backend/scripts/django-run

```diff
@@ -7,4 +7,4 @@
 cd /app/backend/
 
 # Run Daphne
-exec /usr/local/bin/daphne -b 0.0.0.0 -p 80 pennclubs.asgi:application
+exec /usr/local/bin/uvicorn --host 0.0.0.0 --port 80 --workers 3 pennclubs.asgi:application
```

Comment:
Looking at the Uvicorn page, looks like it mentions running uvicorn through gunicorn to manage processes:

https://www.uvicorn.org/deployment/

No anchor tag, but scroll down to "using a process manager".